src/auth-service/utils/email.msgs.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
       `Dear ${firstName + " " + lastName} \n\n` +
```

Comment:
Not sure but dont we need a comma (,) after lastName @Baalmart?

Reply:
Resolved. @ja-odur